src/typings.ts

```diff
@@ -175,7 +365,9 @@ export interface Token {
   access_token: string;
   refresh_token: string;
   token_type: string;
+  /** created at in seconds */
```

Comment:
"in seconds" is cryptic for a date/time value.
perhaps mention that it's a unix timestamp in seconds?